iroh-gossip/src/net.rs

```diff
@@ -345,10 +348,14 @@ struct Actor {
 
 impl Actor {
     pub async fn run(mut self) -> anyhow::Result<()> {
+        let mut i = 0;
```

Comment:
❤️  I know the feeling so well when you have to add a counter to your actor loops